lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -1,52 +1,58 @@
 /**
  * View for a topic card. Displays a Topic.
  */
-;(function (define) {
+;(function(define) {
     'use strict';
-    define(['backbone', 'underscore', 'gettext', 'js/components/card/views/card'],
-        function (Backbone, _, gettext, CardView) {
+    define([
+            'backbone', 'underscore', 'gettext', 'js/components/card/views/card',
+            'edx-ui-toolkit/js/utils/html-utils'
+        ],
+        function(Backbone, _, gettext, CardView, HtmlUtils) {
             var TeamCountDetailView = Backbone.View.extend({
                 tagName: 'p',
                 className: 'team-count',
 
-                initialize: function () {
+                initialize: function() {
                     this.render();
                 },
 
-                render: function () {
-                    var team_count = this.model.get('team_count');
-                    this.$el.html(_.escape(interpolate(
-                        ngettext('%(team_count)s Team', '%(team_count)s Teams', team_count),
-                        {team_count: team_count},
-                        true
-                    )));
+                render: function() {
+                    var teamCount = this.model.get('team_count');
+                    this.$el.html(HtmlUtils.interpolateHtml(
```

Comment:
This example highlights two oddnesses:
- If we write the rule that you only need `HtmlUtils.interpolateHtml()` when used in conjunction with `HtmlUtils.HTML()` for mixing plain text and HTML, then here we would use `HtmlUtils.escape(StringUtils.interpolate(...))`.  
  -- Do we want to have people use `HtmlUtils.interpolateHtml()` as a shortcut for the above as you have done here?
- This also creates the odd code where `toString()` is required.  However, you we do in fact need `interpolateHtml` with a mix of plain text and HTML, we'd still need to either use `toString()` or the odd combo of `HtmlUtils.escape(HtmlUtils.interpolateHtml(...)).
- The last option would be to have an `HtmlUtils.jQueryHtml()` of some such name that is a synonym for `HtmlUtils.escape()` for the above case.